types.go

```diff
@@ -0,0 +1,120 @@
+package fq
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TODO(aaron-prindle) currently testing with one concurrent request
+const C = 1 // const C = 300
+
+// TODO(aaron-prindle) currently service time "G" is not implemented entirely
+const G = 100 // const G = 60000
+
+type Packet struct {
+	// request   http.Request
+	item interface{}
+	// virfinish uint64
+	size  uint64
+	queue *Queue
```

Comment:
Instead of `queue *Queue` we could have `queueIndex int`.

Reply:
I will update this